api/src/modules/custom-projects/custom-projects.controller.ts

```diff
@@ -145,25 +144,24 @@ export class CustomProjectsController {
   @TsRestHandler(customProjectContract.updateCustomProject)
   async updateCustomProject(
     @GetUser() user: User,
-    @Body(new ValidationPipe({ enableDebugMessages: true, transform: true }))
-    dto: Partial<CustomProject>,
   ): Promise<ControllerResponse> {
     return tsRestHandler(
       customProjectContract.updateCustomProject,
-      async ({ params: { id } }) => {
+      async ({ params: { id }, body }) => {
         if (
           !(await this.customProjects.areProjectsCreatedByUser(user.id, [id]))
         ) {
-          return {
-            status: 401,
-            body: null,
-          };
+          throw new ForbiddenException();
         }
-
-        const updatedEntity = await this.customProjects.update(id, dto);
```

Comment:
This was the bug, this is using nestjs-base-service's update method, which will perform a classic UPDATE, but won't run any re-computation, which will be potentially the goal of editing a custom project